<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Maze Search Algorithm</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSearch">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionSearchDFS"/>
    <addaction name="actionSearchBFS"/>
    <addaction name="actionSearchBestFirstSearch"/>
    <addaction name="actionReset"/>
    <addaction name="actionComplexity"/>
   </widget>
   <addaction name="menuSearch"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSearchDFS">
   <property name="text">
    <string>Depth First Search</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset Maze</string>
   </property>
  </action>
  <action name="actionSearchBFS">
   <property name="text">
    <string>Breadth First Search</string>
   </property>
  </action>
  <action name="actionSearchBestFirstSearch">
   <property name="text">
    <string>Best First Search</string>
   </property>
  </action>
  <action name="actionComplexity">
   <property name="text">
    <string>Change Maze Complexity</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
